CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 
 - Optimized the *Object Instancer* node.
 - Make all numeric types comparable.
+- Added Mask Layer to the Set GP Layer Attributes node, GP Object Input node, GP Layer Info node, GP Layer From Frames node, GP Object Output node
```

Comment:
Actually, we are in the same subversion, so we shouldn't update the CHNAGELOG. My apologies, this should be removed.